app/javascript/packages/document-capture/components/in-person-post-office-outage-alert.tsx

```diff
@@ -0,0 +1,23 @@
+import { t } from '@18f/identity-i18n';
+import { Alert } from '@18f/identity-components';
+import { getConfigValue } from '@18f/identity-config';
+
+function InPersonPostOfficeOutageAlert() {
```

Comment:
could we call it `InPersonUspsOutageAlert` instead?

Reply:
Sure